consider adding a `main` function to `src\main.rs`

use ewasm_api::prelude::*;

fn entry() {
    let a: Hash = block_hash(1);
    finish_data(&a.bytes);
}

ewasm_entry_point!(entry);


error[E0601]: `main` function not found in crate `myrust`
  |
  = note: consider adding a `main` function to `src\main.rs`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0601`.

--- a/README.md
+++ b/README.md
@@ -13,6 +13,12 @@ Add the dependency, as usual:
 ewasm-api = "0.10"
 ```
 
+Make sure the project is a library of `cdylib` type:
+```toml
+[lib]
+crate-type = ["cdylib"]
+```
+
 In your project, include the prelude:
 ```rust
 use ewasm_api::prelude::*;

